.gitignore

```diff
@@ -25,3 +25,10 @@ deltachat-ffi/html
 deltachat-ffi/xml
 
 .rsynclist
+
+# ignore backup files
+*.bak
+
+# ignore test-data
+/test-data
+
```

Comment:
to ignore files only specific to your local setup, you can also use `~/.gitignore` (in your private home directory)

Reply:
Ok, that's a good idea. Thanks for that hint.

But what about `/test-data` ? Should it be excluded from `jni/deltachat-core-rust.gitignore` ?

Reply:
no. `/test-data` are needed to run the test eg. in ci. it contains test-images, messages and so on.